pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/gophercloud/gophercloud/openstack/loadbalancer/v2/loadbalancers"
 	"github.com/gophercloud/gophercloud/openstack/loadbalancer/v2/monitors"
 	"github.com/gophercloud/gophercloud/openstack/loadbalancer/v2/pools"
+	"github.com/gophercloud/gophercloud/openstack/loadbalancer/v2/providers"
```

Comment:
I think you need to regenerate `dep` after that.

Reply:
yep! I'll update now that gophercloud side got merged